sentry-android-core/src/main/java/io/sentry/android/core/internal/util/SentryFrameMetricsCollector.java

```diff
@@ -34,8 +35,8 @@ public final class SentryFrameMetricsCollector implements Application.ActivityLi
   private final @NotNull SentryOptions options;
   private @Nullable Handler handler;
   private @Nullable WeakReference<Window> currentWindow;
-  private final @NotNull HashMap<String, FrameMetricsCollectorListener> listenerMap =
-      new HashMap<>();
+  private final @NotNull Map<String, FrameMetricsCollectorListener> listenerMap =
```

Comment:
`h`: I see that `trackedWindows` is being access via `stopCollection()` as well - I think this needs to be thread-safe too right?

Reply:
yeah, nice catch